### PR TITLE
CC-14636 UnzerGui module enabler

### PR DESCRIPTION
**Pull request by kkicheglovspryker**
Ticket: https://spryker.atlassian.net/browse/CC-14636

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# UnzerGui Module
+[![CI](https://github.com/spryker-eco/unzer-gui/actions/workflows/ci.yml/badge.svg)](https://github.com/spryker-eco/unzer-gui/actions/workflows/ci.yml)
+[![Latest Stable Version](https://poser.pugx.org/spryker-eco/unzer-gui/v/stable.svg)](https://packagist.org/packages/spryker-eco/unzer-gui)
+[![License](https://img.shields.io/github/license/spryker-eco/unzer-gui.svg?b=master)](https://github.com/spryker-eco/unzer-gui)
+[![Scrutinizer Code Quality](https://scrutinizer-ci.com/g/spryker-eco/unzer-gui/badges/quality-score.png?b=master)](https://scrutinizer-ci.com/g/spryker-eco/unzer-gui/?branch=master)
+[![Build Status](https://scrutinizer-ci.com/g/spryker-eco/unzer-gui/badges/build.png?b=master)](https://scrutinizer-ci.com/g/spryker-eco/unzer-gui/build-status/master)
+[![Minimum PHP Version](https://img.shields.io/badge/php-%3E%3D%207.3-8892BF.svg)](https://php.net/)
+## Installation
```
Comment (AsonUnique) [inline README.md:11]: Please, add small description of the module here.

File: phpstan.neon [hunk 2]
```diff
@@ -0,0 +1,17 @@
+parameters:
+    level: 8
+    checkMissingIterableValueType: false
+    excludes_analyse:
+
+    bootstrapFiles:
```
Comment (AsonUnique) [inline phpstan.neon:6]: Please, check if you need this

File: tooling.yml [hunk 3]
```diff
@@ -0,0 +1,2 @@
+architecture-sniffer:
```
Comment (AsonUnique) [inline tooling.yml:1]: please, add code-sniffer too

File: README.md (hunk 1 above)
Comment (kkicheglovspryker) [reply README.md:11]: done

File: phpstan.neon (hunk 2 above)
Comment (kkicheglovspryker) [reply phpstan.neon:6]: need for future tests inside github actions

File: tooling.yml (hunk 3 above)
Comment (kkicheglovspryker) [reply tooling.yml:1]: done

File: phpstan.neon (hunk 2 above)
Comment (AsonUnique) [reply phpstan.neon:6]: Do you plan to add some acceptance tests?

**Review by AsonUnique — APPROVED** [review]

**Pull request merged**